test/math.test.ts

```diff
@@ -101,10 +103,49 @@ test('rounding a date to the closest minute', () => {
   expect(date5.getSeconds()).toBe(0);
 });
 
-test('adding days to date', () => {
-  const date1 = new Date(2015, 10, 1, 0, 0, 0);
-  expect(date1.getDate()).toBe(1);
+describe('test adding days to date on different time-zones', () => {
+  const testTimezone = (tz: string) =>
+    describe('timezone: ' + tz, () => {
+      beforeAll(() => {
+        timezoneMock.register('Brazil/East');
```

Comment:
it looks like all these tests are in "Brazil/East" so why is the timezone string being passed in using the array below?

Reply:
you are right, I forgot to use tz, fixed